settings.gradle.kts

```diff
@@ -6,6 +6,9 @@ dependencyResolutionManagement {
         maven("https://repo.opencollab.dev/maven-snapshots/") {
             name = "opencollab-snapshots"
         }
+        maven("https://s01.oss.sonatype.org/content/repositories/snapshots/") {
+            name = "sonatype-snapshots"
+        }
```

Comment:
keep the comment about removing this repo when adventure snapshot is no longer used